static/app/views/performance/traceDetails/index.tsx

```diff
@@ -147,6 +147,7 @@ class TraceSummary extends Component<Props> {
 
     return (
       <TraceFullDetailedQuery
+        detailed
```

Comment:
Can we instead have a type="detailed" or type="spans"? The dual prop is unnecessary